src/main/java/ua/okwine/productexpirationdate/requestWrappers/ProductRequest.java

```diff
@@ -0,0 +1,99 @@
+package ua.okwine.productexpirationdate.requestWrappers;
+
+import ua.okwine.productexpirationdate.entity.Product;
+import ua.okwine.productexpirationdate.entity.Supplier;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+public class ProductRequest {
+
+    private Product product;
+    private double quantity = 0;
+
+    public ProductRequest() {
+        this.product = new Product();
+    }
+
+    public ProductRequest(UUID id, String vendorCode, String barCode, String productName,
+                          LocalDate produce, LocalDate expirationDate, Supplier supplier, int quantity) {
+        this.product = new Product(id, vendorCode, barCode, productName, produce, expirationDate, supplier);
+        this.quantity = quantity;
+    }
```

Comment:
I saw a constructor in the end of the class and here it's in the beginning, it's better to have a standard so other people would expect to see them at one place in any class